### PR TITLE
feat: alert message template - pt1

### PR DESCRIPTION
**Pull request by wrn14897**

### CODE REVIEW

**Review by MikeShi42 — DISMISSED** [review]

File: packages/api/src/tasks/__tests__/checkAlerts.test.ts [hunk 1]
```diff
@@ -404,14 +373,14 @@ describe('checkAlerts', () => {
...
         {
-          text: 'Alert for "Max Duration" in "My Dashboard" - 102 exceeds 10',
           blocks: [
             {
               text: {
                 text: [
                   `*<http://localhost:9090/dashboards/${dashboard._id}?from=1700170200000&granularity=5+minute&to=1700174700000 | Alert for "Max Duration" in "My Dashboard">*`,
                   'Group: "HyperDX"',
                   '102 exceeds 10',
+                  '',
```
Comment (MikeShi42) [inline packages/api/src/tasks/__tests__/checkAlerts.test.ts:383]: nit: should we just `trim()` the rendered string so we don't send extra new lines?

File: packages/api/src/routers/external-api/v1/alerts.ts [hunk 2]
```diff
@@ -54,53 +53,6 @@ const validateGroupBy = async (
   next();
 };
 
-const translateExternalAlertToInternalAlert = (
```
Comment (wrn14897) [inline packages/api/src/routers/external-api/v1/alerts.ts:57]: Move these two methods to `zod.ts`

File: packages/api/package.json [hunk 3]
```diff
@@ -93,7 +93,7 @@
   "scripts": {
     "start": "node ./build/index.js",
     "dev": "nodemon --signal SIGTERM -e ts,json --exec 'ts-node' --transpile-only -r tsconfig-paths/register -r '@hyperdx/node-opentelemetry/build/src/tracing' ./src/index.ts",
-    "dev:task": "ts-node -r tsconfig-paths/register -r '@hyperdx/node-opentelemetry/build/src/tracing' ./src/tasks/index.ts",
+    "dev:task": "ts-node -r tsconfig-paths/register ./src/tasks/index.ts",
```
Comment (wrn14897) [inline packages/api/package.json:96]: disable tracing for the scheduled task

**Review by MikeShi42 — DISMISSED** [review]

File: packages/api/src/tasks/__tests__/checkAlerts.test.ts (hunk 1 above)
Comment (wrn14897) [reply packages/api/src/tasks/__tests__/checkAlerts.test.ts:383]: good idea. let me do it in the next pr

**Pull request merged**